README.md

```diff
@@ -0,0 +1,6 @@
+# HSS1394
+
+*High-speed MIDI-over-Firewire device access library for Windows and macOS.*
+
+HSS1394 is a high-speed MIDI-over-Firewire protocol developed by Stanton Magnetics, Inc.
+This project contains the LGPLv3 library code that interfaces with HSS1394 devices on Windows and macOS, such as the Stanton SCS.1 controller series.

```

Comment:
This library is maintained by the [Mixxx DJ Software](https://mixxx.org/) team. Contributions are welcome by opening pull requests and issues on [GitHub](https://github.com/mixxxdj/hss1394). If you want to discuss anything about this library with us, please get in touch on our [Zulip chat](https://mixxx.zulipchat.com/).

Reply:
## Building

HSS1394 uses the [CMake build system](https://cmake.org/) and follows standard CMake procedures. From the top directory of this source code repository, run:

```
cmake -DCMAKE_INSTALL_PREFIX=/path/to/install/to -S . -B build
cmake --build build
cmake --install build
```